src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -436,15 +436,17 @@
             },
             "preview": {
                 "label": "Preview",
-                "tooltip": "For every programming exercise, repositories and build plans will be generated. Here you can find their generated names based on the short name of the exercise (and course)",
+                "tooltip": "For every programming exercise, repositories and build plans will be generated. Here, you can find their generated names based on the short name of the exercise (and course)",
```

Comment:
Added a comma after "Here", that's why there is no change in the german translation